### ISSUE TITLE
Photo-Relistic Single Image Super-Resolution Using a Generative Adversarial Network

### ISSUE THREAD

**Issue by dltmddbs100** [dltmddbs100/Paper-log#10]
## Main Idea
**Capture perceptual reality in super-resolution with SRGAN**
Paper: <https://arxiv.org/abs/1609.04802>

## Abstract
Despite the breakthroughs in accuracy and speed of single image super-resolution using faster and deeper convolutional neural networks, one central problem remains largely unsolved: how do we recover the finer texture details when we super-resolve at large upscaling factors? The behavior of optimization-based super-resolution methods is principally driven by the choice of the objective function. Recent work has largely focused on minimizing the mean squared reconstruction error. The resulting estimates have high peak signal-to-noise ratios, but they are often lacking high-frequency details and are perceptually unsatisfying in the sense that they fail to match the fidelity expected at the higher resolution. In this paper, we present SRGAN,
a generative adversarial network (GAN) for image superresolution (SR). To our knowledge, it is the first framework capable of inferring photo-realistic natural images for 4×upscaling factors. To achieve this, we propose a perceptual loss function which consists of an adversarial loss and a content loss. The adversarial loss pushes our solution to the natural image manifold using a discriminator network that is trained to differentiate between the super-resolved images and original photo-realistic images. In addition, we use a content loss motivated by perceptual similarity instead of similarity in pixel space. Our deep residual network
is able to recover photo-realistic textures from heavily downsampled images on public benchmarks. An extensive
mean-opinion-score (MOS) test shows hugely significant gains in perceptual quality using SRGAN. The MOS scores
obtained with SRGAN are closer to those of the original high-resolution images than to those obtained with any
state-of-the-art method.

## Contribution
+ Super resolution task에서 mean squared reconstruction error의 한계를 극복, GAN을 이용한 새로운 고해상도 모델인 SRGAN 제안
+ MSE에 기반을 둔 PSNR이나 SSIM과 같은 metric의 한계를 극복 => adversarial loss와 content loss로 구성된 새로운 Perceptual loss function 구축
+ Pretrained된 VGG19 network의 feature map의 euclidean distance 측정, content loss function으로 정의
+ 상위 층의 feature map에서 정의된 loss를 적용했을 때 더 좋은 질감의 detail 보존
+ SR은 고정된 정답이 있는 것이 아니라, 가능한 여러가지의 solution들을 가짐, MSE 방식은  solution들을 pixel 단위로 평균을 내어 산출해 detail의 손실이 큼
+ 반면 SRGAN은 한 가지 solution에 집중해 이미지를 생성, 비록 원본 이미지에서 나타내는 detail의 모양과 완벽하게 부합하지는 않을지라도 고화질의 자연스러운 이미지를 생성

## Reviews
<https://deepkerry.tistory.com/20>
